priv/repo/migrations/20220915094102_add_foreign_references_and_delete_behaviors_to_tables.exs

```diff
@@ -2,15 +2,13 @@ defmodule Ret.Repo.Migrations.AddForeignReferencesAndDeleteBehaviorToTables do
   use Ecto.Migration
```

Comment:
Hmm.. Is it ok to edit a migration like this? At first I thought it was just formatting changes but looks like there might be others. Aren't migrations immutable?

Reply:
You are correct, migrations should not be functionally changed. In this case the change is a refactor, the behavior remains the same. More importantly though, this migration hasn’t yet been merged into `master`.